scripts/west_commands/genboard/templates/nrf52/board_twister.yml.jinja2

```diff
@@ -1,8 +1,4 @@
-{% if ncs_version < hwmv2_since %}
-identifier: {{ board }}
-{% else %}
 identifier: {{ board }}/{{ soc }}
-{% endif %}
 name: {{ board_desc }}
 vendor: {{ vendor }}
```

Comment:
n.b. there is some new field here which Ben has added, just fyi

Reply:
has it trippled down to NCS?